inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/user/TenantRolePageRequest.java

```diff
@@ -38,8 +38,8 @@ public class TenantRolePageRequest extends PageRequest {
     @ApiModelProperty(value = "Role code")
     private String roleCode;
 
-    @ApiModelProperty(value = "Tenant")
-    private String tenant;
```

Comment:
Why delete it？

Reply:
This interface should not support multiple tenant,  **_keyword_** is use to fuzzy match the tenant.